graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/TypeUtils.kt

```diff
@@ -146,6 +146,30 @@ class TypeUtils(private val packageName: String, private val config: CodeGenConf
         }
     }
 
+    fun findInnerType(fieldType: Type<*>, useInterfaceType: Boolean = false, useWildcardType: Boolean = false): TypeName {
```

Comment:
The method is none-trivial, should we add some docs to it?

Reply:
Will do.